.cirrus.yml

```diff
@@ -44,6 +44,12 @@ publishing_binary_task_template: &PUBLISHING_BINARY_TASK_TEMPLATE
   << : *BUNDLER_CACHE_TEMPLATE
   << : *RUN_SETUP_SCRIPT_TEMPLATE
 
+task:
+  name: Development
+  << : *PUBLISHING_BINARY_TASK_TEMPLATE
```

Comment:
I think you wanted non-manual trigger here, and also make sure the branch is develop

Reply:
that's why I put it in github action